components/802.15.4_RF/stm-s2lp-rf-driver/stm-s2lp-rf-driver/NanostackRfPhys2lp.h

```diff
@@ -28,7 +28,7 @@
 // Uncomment to use testing gpios attached to TX/RX processes
 // #define TEST_GPIOS_ENABLED
 
-#if defined(TARGET_MTB_STM_S2LP)
+#if defined(TARGET_MTB_STM_S2LP) || defined(TARGET_MTB_STM_S2LP_CT)
 #if !defined(S2LP_SPI_SDI)
```

Comment:
I believe we shouldn't add custom configuration/defines to the drivers.
Instead, please use mbed_app.json to configure pin config in the application (this will let us remove target dependencies in Mbed OS).
See example for [esp8266-driver](https://github.com/ARMmbed/mbed-os/tree/master/components/wifi/esp8266-driver)

Reply:
@MarceloSalazar , can configuration/define change be done in another PR? 
We would like to enable custom target first to verify we get target to our CI correctly. When that phase is completed we could continue with additional optimisation.

Reply:
You certainly can, but we need to move away from custom config in drivers asap and this file will be modified again. Every PR requires review, testing and time ;-)

@0xc0170 let us know if you think we can proceed as it's

Reply:
As it is already there , lets get this in and will be fixed - @artokin can you create a tracking issue for this ?

Reply:
Sure, the issue is: https://jira.arm.com/browse/IOTTHD-4097